docs/conf.py

```diff
@@ -24,21 +22,17 @@
 # -- Project information - these are special values used by sphinx. -------
 
 from dwaveoceansdk import __version__ as version
-from dwaveoceansdk import __version__ as release
 
 setup_cfg = configparser.ConfigParser()
 setup_cfg.read(os.path.join(sdk_directory, 'setup.cfg'))
 
-author = setup_cfg['metadata']['author']
-copyright = setup_cfg['metadata']['author']
+python_requires = setup_cfg['options']['python_requires']
 
-project = 'Ocean Documentation'
+copyright = 'D-Wave Quantum Inc' #setup_cfg['metadata']['author']
 
-# Also add our own 'special value', the minimum supported Python version
 rst_prolog = f"""
-.. |python_requires| replace:: {setup_cfg['options']['python_requires']}
-.. |array-like| replace:: array-like
-.. _array-like: https://numpy.org/devdocs/glossary.html#term-array_like
+.. |python_requires| replace:: {python_requires}
+.. |sdk_version| replace:: {version}
```

Comment:
This causes errors and warnings for me when building:
```
<rst_prolog>:3: ERROR: Error in "replace" directive: may contain a single paragraph only.
<rst_prolog>:3: WARNING: Substitution definition "sdk_version" empty or invalid.
```

Reply:
Yeah, I haven't figured that one out yet. It does its job properly but generates that warning. It might be some preprocessing run or something.